docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -116,7 +116,7 @@ In addition there is _`StaticRootPath`_ which can be set as a built-in at build
 
 ### Repository - Editor (`repository.editor`)
 
-- `LINE_WRAP_EXTENSIONS`: **.txt,.md,.markdown,.mdown,.mkd,**: List of file extensions for which lines should be wrapped in the Monaco editor. Separate extensions with a comma. To line wrap files without an extension, just put a comma
+- `LINE_WRAP_EXTENSIONS`: **.txt,.md,.markdown,.mdown,.mkd,.livemd,**: List of file extensions for which lines should be wrapped in the Monaco editor. Separate extensions with a comma. To line wrap files without an extension, just put a comma
```

Comment:
That is a different setting: `[repository.editor].LINE_WRAP_EXTENSIONS` vs `[markdown].FILE_EXTENSIONS`.
There's rather the question: Should the former setting also be line wrapped?

Reply:
Yes, line wrapping makes sense. The Livebook also wraps lines. 

I think it also improves the user experience. Thanks for asking